### PR TITLE
[new] fix .env.example missleading names and networks

### PR DESCRIPTION
**Pull request by GustavoRSSilva**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -26,6 +26,15 @@ IAB_KOVAN_ADDRESS=address_iab_kovan
 IAB_MAINNET_ADDRESS=0xb015D9bBabc472BBfC990ED6A0C961a90a482C57
```
Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

File: migrations/4_deploy_advertisement.js [hunk 2]
```diff
@@ -9,6 +10,9 @@ module.exports = function(deployer, network) {
     switch (network) {
         case 'development':
             AppCoins.deployed()
+            .then(function() {
+                return  deployer.deploy(CampaignLibrary);
```
Comment (tcrosado) [inline migrations/4_deploy_advertisement.js]: What is the difference between pr #62 and this one? 
It seams like this pull request is similar with a bit more code on configurations.

Using this deploy we can later deploy a new advertisement storage without needing to deploy CampaingLibrary if there is no change in the latter one right?

Comment (GustavoRSSilva) [reply migrations/4_deploy_advertisement.js]: In order for the etherscan to recognize the code, i added the campaign library there. It is a  helper: can be found here (the code) https://ropsten.etherscan.io/address/0x251da5d3238084f40446d358335ee878360c2e9d#code

File: .env.example (hunk 1 above)
Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (GustavoRSSilva) [reply .env.example:26]: you are right merging both PR.

**Pull request merged**